_archive/mv2/api/nativeMessaging/host/install_host.sh

```diff
@@ -6,14 +6,14 @@
 set -e
 
 DIR="$( cd "$( dirname "$0" )" && pwd )"
-if [ "$(uname -s)" = "Darwin" ]; then
-  if [ "$(whoami)" = "root" ]; then
+if [ $(uname -s) == 'Darwin' ]; then
+  if [ "$(whoami)" == "root" ]; then
     TARGET_DIR="/Library/Google/Chrome/NativeMessagingHosts"
   else
     TARGET_DIR="$HOME/Library/Application Support/Google/Chrome/NativeMessagingHosts"
   fi
 else
-  if [ "$(whoami)" = "root" ]; then
+  if [ "$(whoami)" == "root" ]; then
```

Comment:
Under the root instance to ensure that the script is accessible by all users, set the `native-messaging-example-host` file to execute.

`chmod a+x "$DIR/native-messaging-example-host"`